Dockerfile.dev

```diff
@@ -19,5 +19,7 @@ RUN mv /poktroll/bin/poktrolld /usr/bin/poktrolld
 
 EXPOSE 8545
```

Comment:
NIT: Adding a comment on what each exposed port is used for will go a long way!

Reply:
@Olshansk these will be changed soon, I'll prepare a doc for us to discuss the configs/ports and dependencies for different actors.